### PR TITLE
7.x wcag

### PR DESCRIPTION
**Pull request by MorganDawe**
**JIRA Ticket**: (https://jira.duraspace.org/browse/ISLANDORA-2217)

#WCAG Islandora Compound Object Solution Pack

aXe accessibility engine: https://www.deque.com/axe/
Web Accessibility inititive: https://www.w3.org/WAI/intro/wcag

# What does this Pull Request do?

The goal of this pull request is to bring the compound object solution pack into compliance with WCAG 2.0 standards.

It is important to note that this pull aims to address accessibility violations as seen by the average, non authenticated users. Administrative pages remain unaffected.
# What's new?
Adds alt attributes to the parent thumbnail link if available and an alt tag to the sibling thumbnail links.

# How should this be tested?

The easiest way to test these changes locally:
- install the aXe browser plugin in your browser of choice (found here: https://www.deque.com/axe/).
- Recommended: Enable the ‘AT Subtheme’ found in the Adaptive Core (https://www.drupal.org/project/adaptivetheme). This theme is the closest to matching wcag 2.0 requirements i could find.
- Run the aXe tool on a compound object page, or another wcag 2.0 compliance testing toolkit, and examine the page for WCAG 2.0 violations sourced from this modules code.

# Interested parties
Tag (@DiegoPino @Islandora/7-x-1-x-committers)

### CODE REVIEW

**Comment by DiegoPino** [comment]
@MorganDawe same as the other pull, will test during the week. Thanks

File: theme/islandora-compound-prev-next.tpl.php [hunk 1]
```diff
@@ -32,7 +32,9 @@
        theme_image(
          array(
            'path' => $parent_tn,
-           'attributes' => array(),
+           'attributes' => array(
+              'alt' => $parent_label,
```
Comment (qadan) [inline theme/islandora-compound-prev-next.tpl.php]: Coder is upset at the indentation level here

**Comment by qadan** [comment]
Had to re-run the build, but things are passing 👍

**Review by qadan — APPROVED** [review]

File: theme/islandora-compound-prev-next.tpl.php [hunk 2]
```diff
@@ -61,7 +63,10 @@
        theme_image(
          array(
            'path' => $sibling['TN'],
-           'attributes' => array('class' => $sibling['class']),
```
Comment (DiegoPino) [inline theme/islandora-compound-prev-next.tpl.php:64]: Should the Link l() have a "title" attribute?

**Review by DiegoPino — CHANGES_REQUESTED** [review]
Title in l()

File: theme/islandora-compound-prev-next.tpl.php (hunk 2 above)
Comment (MorganDawe) [reply theme/islandora-compound-prev-next.tpl.php:64]: Good catch. I have added the 'title' and 'alt' attributes to the theme image. Need to update the l() function now.

Comment (MorganDawe) [reply theme/islandora-compound-prev-next.tpl.php:64]: Updated the l() function with a title.

**Review by DiegoPino — APPROVED** [review]

**Comment by DiegoPino** [comment]
@MorganDawe looks good now. Merging. Sadly i have no time right now to test the other ones. @Islandora/7-x-1-x-committers mind lending a pair of eyes to get other WACG ones tested? Thanks

**Pull request merged**